Contents/DefaultPrefs.json

```diff
@@ -13,6 +13,11 @@
 	"label": "Sort artist by name [Last, First]",
 	"type": "bool",
 	"default": "true"
+},{
+	"id": "simplify_title",
+	"label": "Simplify the books' titles",
```

Comment:
There might be a more descriptive way to label this for the user, that doesn't require them to read the docs to know what it does?

Reply:
Hmm, I'm not really sure what a good alternative would be, do you have any ideas?  "Remove extra text from book titles"?

It could be "Don't include subtitles in book titles" but thats not all the setting does.

Reply:
What do you think about: `Simplify book titles (remove subtitle, series, abridged, etc)`?

Reply:
Yeah that sounds good! Or a slight tweak, how about `Simplify book titles (remove subtitle, series part, (un)abridged, etc.)`?  This seems to describe what it's doing a bit more precisely.

Here's what it looks like in the settings window so you can get an idea of how much space it's taking up:

<img width="748" alt="image" src="https://user-images.githubusercontent.com/9214195/181080101-89a5e19f-7406-4080-bd4a-75390b93e394.png">